docs/advanced-topics/kubernetes-dashboard.md

```diff
@@ -0,0 +1,134 @@
+# Kubernetes Dashboard Access and Authentication
+
+## Default Setup
+
+    kubernetesDashboard:
+      adminPrivileges: true
+      insecureLogin: false
+
+In the default setup the Dashboard is configured using the `--auto-generate-certificates` flag and has Admin privileges
+
+###### Access the dashboard  using `kubectl proxy` command:
+
+    http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/
+
+
+## Expose the Dashboard using a ELB with self-signed certificates
+
+kubernetesDashboard:
+  adminPrivileges: false
+  insecureLogin: false
+
+Ex.
+
+    apiVersion: v1
+    kind: Service
+    metadata:
+      annotations:
+        external-dns.alpha.kubernetes.io/hostname: "kubedash.example.com"
+      name: kubernetes-dashboard
+      labels:
+        run: kubernetes-dashboard
+      namespace: kube-system
+    spec:
+      type: LoadBalancer
+    # uncomment if you want to restrict the access to allowed IP's
+    #  loadBalancerSourceRanges:
+    #  - x.x.x.x/32
+      ports:
+      - port: 443
+        targetPort: 8443
+        protocol: TCP
+      selector:
+        k8s-app: kubernetes-dashboard   
+###### Access the dashboard  using `kubectl proxy` command:
+
+    http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/
+
+
+## Expose the Dashboard using a ELB with trusted certificates
+
+    kubernetesDashboard:
+      adminPrivileges: false
+      insecureLogin: true
+
+Ex.
+
+    apiVersion: v1
+    kind: Service
+    metadata:
+      annotations:
+        service.beta.kubernetes.io/aws-load-balancer-ssl-cert:
+        #replace with your certificate ARN
+        arn:aws:acm:us-east-1:XXXXXXXXXX:certificate/xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxx
+        service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
+        external-dns.alpha.kubernetes.io/hostname: "kubedash.example.com"
+      name: kubernetes-dashboard
+      labels:
+        run: kubernetes-dashboard
+      namespace: kube-system
+    spec:
+      type: LoadBalancer
+      ports:
+      - port: 443
+        targetPort: 9090
+        protocol: TCP
+      selector:
+        k8s-app: kubernetes-dashboard  
+
+###### Access the dashboard  using `kubectl proxy` command:
+
+    http://localhost:8001/api/v1/namespaces/kube-system/services/http:kubernetes-dashboard:/proxy/
```

Comment:
Is `proxy` relevant when exposed via ELB?

Reply:
Sometimes can be useful for debugging. Just wanted to make it clear, but I suppose we can specify it only once, separately.